### PR TITLE
Improve PAS form show template

### PR DESCRIPTION
**Pull request by andycochran**
This PR fixes up the PAS show template.

### CODE REVIEW

File: client/app/components/packages/pas-form/show.hbs [hunk 1]
```diff
@@ -1,715 +1,551 @@
...
-          if
-          (eq @package.pasForm.dcpLegalstreetfrontage "717170004")
-          "Mapped but not Build"
-        }}
+        <span class="cell large-3">
+          {{if (eq @package.pasForm.dcpLegalstreetfrontage 717170000) "Mapped and Build"}}
+          {{if (eq @package.pasForm.dcpLegalstreetfrontage 717170001) "Private Road"}}
+          {{if (eq @package.pasForm.dcpLegalstreetfrontage 717170002) "Record Street"}}
+          {{if (eq @package.pasForm.dcpLegalstreetfrontage 717170003) "Corporation Counsel Opinion"}}
+          {{if (eq @package.pasForm.dcpLegalstreetfrontage 717170004) "Mapped but not Build"}}
```
Comment (allthesignals) [inline client/app/components/packages/pas-form/show.hbs:271]: We neeeeeed that code lookup thing @godfreyyeung is designing :)

File: client/app/components/packages/pas-form/show.hbs [hunk 2]
```diff
@@ -1,715 +1,551 @@
...
-          (eq @package.pasForm.dcpProjectareaindustrialbusinesszone false)
-          "No"
-        }}
+        <span class="cell large-3">
+          {{if (eq @package.pasForm.dcpProjectareaindustrialbusinesszone true) "Yes"}}
+          {{if (eq @package.pasForm.dcpProjectareaindustrialbusinesszone false) "No"}}
+        </span>
       </p>
 
       {{#if (eq @package.pasForm.dcpProjectareaindustrialbusinesszone true)}}
```
Comment (allthesignals) [inline client/app/components/packages/pas-form/show.hbs:316]: I know this is not part of the commit but IDK why we need to see if it's `true`, why not just:

```
{{#if @package.pasForm.dcpProjectareaindustrialbusinesszone}}
{{/if}}
```

Comment (allthesignals) [reply client/app/components/packages/pas-form/show.hbs:316]: ii'd understand if we need to check if something is `false` as opposed to `null`

Comment (andycochran) [reply client/app/components/packages/pas-form/show.hbs:316]: I imagine this was just a result of copy/paste from one that's evaluating codes for "true."

**Pull request merged**